core/src/com/unciv/ui/MultiplayerScreen.kt

```diff
@@ -402,7 +400,22 @@ class EditMultiplayerGameInfoScreen(game: GameInfo, gameName: String, backScreen
             askPopup.open()
         }.apply { color = Color.RED }
 
-        topTable.add(deleteButton)
+        val giveUpButton = "Give up".toTextButton()
+        giveUpButton.onClick {
+            val askPopup = Popup(this)
+            askPopup.addGoodSizedLabel("Are you sure you want to give up?".tr()).row()
+            askPopup.addButton("Yes"){
+                giveUp(game.gameId, gameName, backScreen)
+                askPopup.close()
+            }
+            askPopup.addButton("No"){
+                askPopup.close()
+            }
+            askPopup.open()
+        }.apply { color = Color.RED }
```

Comment:
Separate this to another line, since it's entirely separate from the onClick. I know you can chain them, but it's less immediately readable.